cmd/bundle/validate.go

```diff
@@ -88,3 +78,31 @@ func newValidateCommand() *cobra.Command {
 
 	return cmd
 }
+
+func prepareBundleForValidate(cmd *cobra.Command, includeLocations bool) *bundle.Bundle {
```

Comment:
The name is misleading. In the summary command, we actually prepare the bundle. In this function though we perform the actual validation rather than any prep.

Reply:
`prepareBundleForValidate` Implies we'll perform the validation after this function call. That is not true though.

Reply:
I read "ForValidate" as validate command, not validate process. However, feel free to improve this and summary.go further.